pkg/image/prune/imagepruner.go

```diff
@@ -190,6 +190,7 @@ func addImageStreamsToGraph(g graph.Graph, streams *imageapi.ImageStreamList, al
 
 		glog.V(4).Infof("Adding ImageStream %s/%s to graph", stream.Namespace, stream.Name)
 		isNode := imagegraph.EnsureImageStreamNode(g, stream)
+		imagegraph.EnsureAllImageStreamTagsNode(g, stream)
```

Comment:
Ok. Given all this, can you remove this call from here? Pruning doesn't need it.